### PR TITLE
docs: migrate docs examples to xy-grid

### LINKED ISSUE #11028: Update examples to use XY Grid
In the documentation for Foundation 6.4.1 it starts by saying how XY Grid is the new grid and that Float Grid and and Flex Grid are now legacy features and how XY Grid is enabled by default. Thats great, however, every example in the documentation for every feature still uses the old legacy Float Grid as the grid for the examples. Please update the documentation because this makes the examples unusable in a XY Grid project.

### PR DESCRIPTION
**Pull request by colin-marshall**
Closes #11028. Also updated the CodePens.

### CODE REVIEW

**Review by ncoden — COMMENTED** [review]
The whole documentation still uses the Float Grid. Why only `equalizer.md` ?

Also, I'll see if we can get a CodePen pro account to work on pens together without having fork them & change URLs to make a change.

**Comment by colin-marshall** [comment]
@ncoden are there other elements that use the float grid in the code examples? That's what I was going for first.

**Comment by colin-marshall** [comment]
I now see Abide uses float grid too. I targeted develop on this PR mistakenly. I should probably close this out and start over.

**Comment by ncoden** [comment]
You can search `column`/`columns` in all `.md` files 😉 

Screenshot

**Comment by colin-marshall** [comment]
@ncoden I think I got them all now.

**Comment by ncoden** [comment]
@colin-marshall Thanks, that awesome ! Did you made a visual check of everything ?
You can probably squash too since there is a lot of commits and the PR is clean (no changes requested).

**Comment by colin-marshall** [comment]
I did a visual test for all of them but somebody should probably do a quick double check before merge. I’ll squash the commits and force push the updated branch.

**Comment by DanielRuf** [comment]
Hi @colin-marshall, there are some conflicts that have to be resolved.

File: docs/pages/style-sherpa.md [hunk 1]
```diff
@@ -95,24 +95,27 @@ The boilerplate uses Foundation's Vertical Menu&mdash;one item is made for each
 
 {{{{raw}}}}
 ```handlebars
-<div class="column row"><div class="row collapse">
-  <div class="large-3 medium-4 columns" data-sticky-container>
-    <ul class="vertical menu">
+<div class="row">
```
Comment (DanielRuf) [inline docs/pages/style-sherpa.md]: Shouldn't this also use the new grid?

Comment (colin-marshall) [reply docs/pages/style-sherpa.md]: The example was derived from style sherpa, so the grid needs to be changed on that repo first.

Comment (DanielRuf) [reply docs/pages/style-sherpa.md]: Ah ok. The rest seems to be fine.

**Review by DanielRuf — APPROVED** [review]

**Review by ncoden — APPROVED** [review]
👍 Thank you @colin-marshall

**Pull request merged**